pendulum/tz/timezone.py

```diff
@@ -24,10 +24,10 @@ class Timezone(tzinfo):
     POST_TRANSITION = 'post'
     TRANSITION_ERROR = 'error'
 
-    def __init__(self, name, transitions,
-                 tzinfos,
-                 default_tzinfo_index,
-                 utc_transition_times):
+    def __init__(self, name=None, transitions=(),
+                 tzinfos=(),
+                 default_tzinfo_index=0,
+                 utc_transition_times=[]):
```

Comment:
Good to know, I can change that in a follow-up commit. Let me know what else needs to be changed before it can be merged so I can make a new commit and squash.